src/main/webapp/components/result-forms/route.js

```diff
@@ -91,7 +73,7 @@ const Item = props => {
         </Dialog>
       ) : null}
       <IndexCardItem {...form} onClick={() => setEditing(true)}>
-        <Actions attributes={form} user={user}>
+        <Actions attributes={form}>
```

Comment:
Going to see if I can use context in a future pr which will allow a user to be passed through to the hook in storybook